sqlalchemy_history/unit_of_work.py

```diff
@@ -21,6 +21,9 @@ def __init__(self, manager):
         self.manager = manager
         self.reset()
 
+    def create_version_session(self, session):
+        return sa.orm.session.Session(bind=session.connection(), join_transaction_mode="rollback_only")
```

Comment:
We would also have to handle sqla 1.4 here aswell.

current CI workflow seems to have a bug, where sqla-1.4 is not getting installed in poetry's env, 

can you also fix that?, if the fix breaks other items in CI we can create a seperate issue for it.

Reply:
Tried fixing this by replacing:
```pip install 'sqlalchemy${{ matrix.sqla }}'```
 with
```poetry add "sqlalchemy${{ matrix.sqla }}"```

it is breaking other tests